src/components/summary/_summary.scss

```
@@ -46,6 +51,12 @@ $hub-row-spacing: 1.3rem;
       background: var(--ons-color-errors-tint);
       border-left: 8px solid var(--ons-color-errors);
     }
+
+    &--card {
+      &:not(:last-child){
+        border-bottom: 1px solid var(--ons-color-borders);
```

Colour also needs changing here
```suggestion
        border-bottom: 1px solid var(--ons-color-borders-list);
```